### DISCUSSION TITLE
Erro no python rede.py

### DISCUSSION THREAD

**Discussion by JuanDuarte2409** [rictom/rede-cnpj#31] [General]
Estou com dificuldade na execução desse código pois, quando quando solicito esse comando
PS C:\Users\User> python "C:\Users\User\Download\rede.py"
C:\Users\User\AppData\Local\Microsoft\WindowsApps\python.exe: can't open file 'C:\\Users\\User\\Download\\rede.py': [Errno 
2] No such file or directory
PS C:\Users\User> 
o script me retorna isso

**Comment by rictom** [comment]
Você precisa fazer um clone do projeto na sua máquina. Recomendo você ver as orientações passo-a-passo em https://github.com/rictom/rede-cnpj/blob/master/doc/instalacao_windows.md
